dydx3/helpers/requests.py

```diff
@@ -25,7 +31,11 @@ def request(uri, method, headers=None, data_values={}):
     )
     if not str(response.status_code).startswith('2'):
         raise DydxApiError(response)
-    return response.json() if response.content else '{}'
+
+    if response.content:
+        return Response(response.json(), response.headers)
```

Comment:
response.json doesn't include headers?

Reply:
it doesn't - the assertion in the first `test_public.py` test shows the returned value

` assert resp.data == {'exists': False}`